package.json

```diff
@@ -29,9 +30,22 @@
 		}
 	},
 	"ava": {
+		"extensions": {
+			"ts": "module"
+		},
 		"files": [
-			".built/test/*.mjs"
-		]
+			"test/*.ts"
+		],
+		"nodeArguments": [
+			"--loader=ts-node/esm",
+			"--experimental-specifier-resolution=node"
+		],
+		"nonSemVerExperiments": {
+			"configurableModuleFormat": true
+		},
+		"environmentVariables": {
+			"TS_NODE_FILES": "true"
+		}
```

Comment:
Source: https://github.com/TypeStrong/ts-node#help-my-types-are-missing

Maybe the types can be refactored to follow that new pattern instead of globals.d.ts, but I don’t think so